executor/runtime/docker/config.go

```diff
@@ -0,0 +1,126 @@
+package docker
+
+import (
+	"errors"
+	log "github.com/sirupsen/logrus"
+	"gopkg.in/urfave/cli.v1"
+	"time"
+)
+
+const MinCfsBandwidth = 1000
```

Comment:
use const (
MinCfsBandwidth = 1000
MaxCfsBandwidth = 1000000
)

Reply:
actually, it appears these aren't used outside of this package. they should be locally scoped.